README.md

```diff
@@ -332,6 +336,8 @@ buildctl build ... \
 
 Note that the inline cache is not imported unless [`--import-cache type=registry,ref=...`](#registry-push-image-and-cache-separately) is provided.
 
+Note that only the image config is affected, the layers are not. Therfore there is no significant difference in image size using `inline` cache.
```

Comment:
"Inline cache embeds cache metadata into the image config. The layers in the image will be left untouched compared to the image with no cache information."

Reply:
https://github.com/moby/buildkit/pull/2489/commits/a3831757d32881a0caa1bd18fbbb0cdd7acf8bef